Remotion/Web/Core/Res/HTML/ExecutionEngine.js

```diff
@@ -70,7 +70,10 @@ function WxePage_Context(
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('hasSubmitted', hasSubmitted);
     ArgumentUtility.CheckNotNullAndTypeIsBoolean('isCached', isCached);
 
-    document.getElementById('wxePostBackSequenceNumberField').setAttribute('value', _postBackSequenceNumber);
+    let dmaWxePostBackSequenceNumberField = document.getElementById('dmaWxePostBackSequenceNumberField');
+
+    if (dmaWxePostBackSequenceNumberField)

```

Comment:
I typicall use an explict null-check in JS, see line 114. It makes the intent more clear when this is not just a sanity check but has semantics associated with it.